src/components/bcros/filing/item/DissolutionVoluntary.vue

```diff
@@ -46,20 +39,7 @@ const dissolutionDateTime =
       <div v-else-if="isStatusCompleted" data-cy="completed-dissolution-details">
         <strong>{{ $t('text.filing.dissolution.completed') }}</strong>
 
-        <p v-if="isEntityFirm">
-          {{ $t('text.filing.dissolution.theStatementOf') }} {{ entityTitle }} {{ currentBusinessName || '' }}
-          {{ $t('text.filing.dissolution.wasSuccessfully') }}&nbsp;{{ $t('text.filing.dissolution.submittedOn') }}&nbsp;
-          <strong>{{ dissolutionDateSubmittedPacific }}</strong>
-          {{ $t('text.filing.dissolution.withDissolutionDateOf') }}&nbsp;<strong>{{ dissolutionDatePacific }}</strong>.
-          {{ $t('text.general.the') }}&nbsp;{{ entityTitle }} {{
-            $t('text.filing.dissolution.hasBeenStruckAndDissolved')
-          }},
-          {{ $t('text.filing.dissolution.ceasedToBe') }}
-          {{ $t('text.filing.dissolution.aRegistered') }}&nbsp;{{ entityTitle }}
-          {{ $t('text.filing.dissolution.underThe') }}&nbsp;{{ actTitle }} Act.
-        </p>
-
-        <p v-if="!isEntityFirm">
```

Comment:
same here

Reply:
brought back but as a v-else instead of a !